diff --git a/dist/src/wallet-connection/queryBankBalances.js b/dist/src/wallet-connection/queryBankBalances.js
index 64cd501f2feb4b12b40d5dd7facfc924a7a7d469..17fde4ded779748229ab4af127543e5b38c094b4 100644
--- a/dist/src/wallet-connection/queryBankBalances.js
+++ b/dist/src/wallet-connection/queryBankBalances.js
@@ -1,5 +1,5 @@
 import { QueryClient, createProtobufRpcClient } from '@cosmjs/stargate';
-import { QueryClientImpl } from 'cosmjs-types/cosmos/bank/v1beta1/query';
+import { QueryClientImpl } from 'cosmjs-types/cosmos/bank/v1beta1/query.js';
 export const queryBankBalances = async (address, tendermint) => {
     const queryClient = new QueryClient(tendermint);
     const rpcClient = createProtobufRpcClient(queryClient);
diff --git a/package.json b/package.json
index 447948800eec6138d4fdada957bcac00e8c938df..026fcb70e191cacea693edfe0602926af12212bc 100644
--- a/package.json
+++ b/package.json
@@ -5,6 +5,7 @@
   "author": "Agoric",
   "version": "0.16.0",
   "main": "dist/index.js",
+  "type": "module",
   "scripts": {
     "prepack": "tsc --build tsconfig.build.json",
     "lint": "eslint . && tsc",
